lib/api/address/v1/service.ts

```diff
@@ -27,20 +27,18 @@ export const searchAddress = async (postCode: string): Promise<SearchAddressResp
       return res;
     });
 
-export const getAddressViaUprn = async (UPRN: string): Promise<SearchAddressResponse> =>
-  axiosInstance
-    .get<AddressAPIResponse>(`${config.addressApiUrlV1}/addresses?uprn=${UPRN}`, {
-      headers: {
-        "skip-x-correlation-id": true,
-      },
-    })
-    .then((res) => ({ addresses: res.data.data.address }))
-    .catch((res) => {
-      if (res.message.toLowerCase().indexOf("network") !== -1) {
```

Comment:
FYI: This part in the catch was copied from elsewhere, but it did not seem to work